packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -51,7 +63,9 @@ const DefaultControlsGroup = ( { items, onClose, toggleItem } ) => {
 	);
 };
 
-const OptionalControlsGroup = ( { items, onClose, toggleItem } ) => {
+const OptionalControlsGroup = ( props: ToolsPanelControlsGroupProps ) => {
+	const { items, onClose, toggleItem } = props;
+
```

Comment:
You should be able to keep using the previous, more compact syntax here:

```suggestion
const OptionalControlsGroup = ( { items, onClose, toggleItem }: ToolsPanelControlsGroupProps ) => {
```